pkg/asset/installconfig/platformcredscheck.go

```diff
@@ -75,6 +76,9 @@ func (a *PlatformCredsCheck) Generate(dependencies asset.Parents) error {
 		if err != nil {
 			return errors.Wrap(err, "testing Engine connection")
 		}
+	case kubevirt.Name:
+		// no creds to check
+		// TODO <nargaman> Add kubeconfig validation
```

Comment:
TODO

Reply:
As we talked, this is not urgent task, will be completed after this PR
Need to call to any API (List Service), the validation should be failed if `Unauthorized` error is returned

Reply:
This is where cluster reachability should be checked.